### ISSUE TITLE
Insufficient color contrast for social media icons in footer

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/alasdoo_com_v2#38]
Social media icons in footer have insufficient color contrast (2.881:1). Minimum color contrast for  graphic elements is 3:1.
